src/history/test/SerializeTests.cpp

```diff
@@ -12,23 +12,40 @@ using namespace stellar;
 
 TEST_CASE("Serialization round trip", "[history]")
 {
-    std::vector<std::string> testFiles = {
-        "stellar-history.testnet.6714239.json",
-        "stellar-history.livenet.15686975.json"};
-    for (auto const& fn : testFiles)
+    std::vector<std::string> inputFiles = {
+        "stellar-history.testnet.6714239.input.json",
+        "stellar-history.livenet.15686975.input.json",
+        "stellar-history.testnet.6714239.networkPassphrase.json"};
+    std::vector<std::string> outputFiles = {
+        "stellar-history.testnet.6714239.output.json",
+        "stellar-history.livenet.15686975.output.json",
+        "stellar-history.testnet.6714239.networkPassphrase.json"};
+    for (int i = 0; i < inputFiles.size(); i++)
     {
         std::string fnPath = "testdata/";
-        fnPath += fn;
-        SECTION("Serialize " + fnPath)
+        std::string inputPath = fnPath + inputFiles[i];
+        std::string outputPath = fnPath + outputFiles[i];
```

Comment:
same here, variable should be called `expectedOutputPath`

Reply:
Done!